lib/spectrum/holding/holding.rb

```diff
@@ -0,0 +1,146 @@
+module Spectrum
```

Comment:
These look like the kinds of refactoring I had been wanting to get to but had put off.

Thanks for fixing this.